src/components/existing-request/existing-request-display.vue

```diff
@@ -303,54 +296,63 @@ export default class ExistingRequestDisplay extends Mixins(
 
-  private get consentDate () {
+  private get consentDate (): string {
+    let ret: string
     if (this.nr.consent_dt) {
-      return Moment(this.nr.consent_dt).tz('America/Vancouver').format('MMMM D[,] YYYY')
+      const date = new Date(this.nr.consent_dt)
+      ret = this.dateToPacificDate(date)
     }
-    return 'Not Yet Received'
+    return ret || 'Not Yet Received'
   }
 
-  private get expiryDate () {
+  private get expiryDate (): string {
+    let ret: string
     if (this.nr.expirationDate) {
-      return Moment(this.nr.expirationDate).tz('America/Vancouver').format('MMMM D[,] YYYY')
+      const date = new Date(this.nr.expirationDate)
+      ret = this.dateToPacificDate(date)
     }
-    return ''
+    return ret || ''
   }
 
-  private get reviewDate () {
+  private get reviewDate (): string {
     if (this.nr.waiting_time) {
-      let queueTime = this.nr.waiting_time
+      let queueTime = this.nr.waiting_time // days
       // get number of days since the nr was submitted
-      let diffDays = Math.abs(this.daysFromToday(this.nr.submittedDate))
-      // get the waiting time by subtracking the current wait time from the amount that we have been waiting
+      const submittedDate = new Date(this.nr.submittedDate)
+      let diffDays = Math.abs(this.daysFromToday(submittedDate))
+      // get the waiting time by subtracting the current wait time from the amount that we have been waiting
       let waitingTime = queueTime - diffDays
       // add the waiting time days to todays date
-      let todaysDate = Moment()
-      let waitingDate = todaysDate.add(waitingTime, 'days')
-      return waitingDate.tz('America/Vancouver')
-        .format('MMMM D[,] YYYY') + ' (' + this.nr.waiting_time + ' days)'
+      let todaysDate = Date.now() // Moment() // *** TODO: fix. This is "now" according to user's local clock/timezone
+      let waitingDate = todaysDate // todaysDate.add(waitingTime, 'days') // *** TODO: fix
+      // return waitingDate.tz('America/Vancouver')
+      //   .format('MMMM D[,] YYYY') + ' (' + this.nr.waiting_time + ' days)'
+      return waitingDate.toString() // *** TODO: fix
     }
-    return ''
+    return 'Unknown'
   }
 
-  private get queueDate () {
+  private get queueDate (): string {
+    let ret: string
     if (this.nr.oldest_draft) {
-      let oldest_draft = Moment(this.nr.oldest_draft)
-      return oldest_draft.tz('America/Vancouver').format('MMMM D[,] YYYY')
+      const date = new Date(this.nr.oldest_draft)
+      ret = this.dateToPacificDate(date)
     }
-    return ''
+    return ret || 'Unknown'
   }
 
-  private get submittedDate () {
+  private get submittedDate (): string {
+    let ret: string
     if (this.nr.submittedDate) {
-      return Moment(this.nr.submittedDate).tz('America/Vancouver').format('MMMM D[,] YYYY, h:mm a') + ' Pacific time'
+      const date = new Date(this.nr.submittedDate)
+      ret = this.dateToPacificDateTime(date)
     }
-    return ''
+    return ret || 'Unknown'
```

Comment:
As these strings display unconditionally in the template, I thought it was better to display 'Unknown' instead of ''.